generate_sl.py

```diff
@@ -59,35 +59,39 @@ def split_and_generate_slurms(sim_dirs, obs_dirs, station_file, rrup_files, outp
         i += max_lines
 
 
 def main():
     parser = argparse.ArgumentParser(description="Prints out a slurm script to run IM Calculation over a run-group")
-    parser.add_argument('-s', '--sim_dir',
+    parser.add_argument('-sim', '--sim_dir',
                         help="Path to sim-run-group containing faults and acceleration in the subfolder */BB/*/*")
-    parser.add_argument('-o', '--obs_dir',
+    parser.add_argument('-obs', '--obs_dir',
                         help="Path to obs-run-group containing faults and accelerations in the subfolder */*/accBB")
     parser.add_argument('-srf', '--srf_dir',
                         help="Path to run-group containing the srf files in the path matching */Srf/*.srf")
     parser.add_argument('-ll', '--station_file',
                         help="Path to a single station file for ruputure distance calculations")
     parser.add_argument('-np', '--processes', default=DEFAULT_N_PROCESSES, help="number of processors to use")
     parser.add_argument('-ml', '--max_lines', default=100, type=int, help="maximum number of lines in a slurm script. Default 100")
-    parser.add_argument('-e', '--extended', action='store_const', const='-e', default='', help="add '-e' to indicate the use of extended pSA period. Default not using")
-    parser.add_argument('-simple', action='store_const',const='-s',default='',
-                        help="Please add '-simple' to indicate if you want to output the big summary csv only(no single station csvs). Default outputting both single station and the big summary csvs")
-    parser.add_argument('-rrup_out_dir', default=DEFAULT_RRUP_OUTDIR, help="output directory to store rupture distances output.Default is {}".format(DEFAULT_RRUP_OUTDIR))
-    
+    parser.add_argument('-e', '--extended_period', action='store_const', const='-e', default='', help="add '-e' to indicate the use of extended pSA period. Default not using")
+    parser.add_argument('-s', '--simple_output', action='store_const',const='-s',default='',
+                        help="Please add '-s' to indicate if you want to output the big summary csv only(no single station csvs). Default outputting both single station and the big summary csvs")
+    parser.add_argument('-o', '--rrup_out_dir', default=DEFAULT_RRUP_OUTDIR, help="output directory to store rupture distances output.Default is {}".format(DEFAULT_RRUP_OUTDIR))
+    parser.add_argument('-fd', '--fd_station_file', help="path to a trimmed station file. eg.'/nesi/nobackup/nesi00213/RunFolder/Cybershake/v18p6/Runs/Kelly/fd_rt01-h0.400.ll'")
+   
     args = parser.parse_args()
     
     if args.srf_dir is not None:
         utils.setup_dir(args.rrup_out_dir)
 
     if args.max_lines <= 0:
         parser.error("-ml argument should come with a number that is 0 < -ml <= (max_lines-header_and_other_prints) allowed by slurm")
-
+    
+    if args.fd_station_file is not None:
+        args.fd_station_file = ' -fd {}'.format(args.fd_station_file)
```

Comment:
Best to not modify part of argparse. Instead create a new variable. Otherwise it's not intuitive how the input gets transformed.